lib/sinon/util/fake_timers.js

```diff
@@ -2,17 +2,34 @@
 
 var llx = require("lolex");
 
-exports.useFakeTimers = function () {
-    var now;
-    var methods = Array.prototype.slice.call(arguments);
+/**
+ *
+ * @param config {number|Date|Object} the unix epoch value to install with (default 0) or
+ */
+exports.useFakeTimers = function (args) {
```

Comment:
So in regards to the config object - will update the documentation.

Next up - the sinonjs website - if I'd like to keep the [table format from lolex](https://github.com/sinonjs/lolex) - would this work with the current page margins of the documentation section on [sinon](http://sinonjs.org/releases/v2.4.0/fake-timers/)? 
It looks about a hundred pixels narrower. Or maybe I should find some other format to display.

Thoughts?